save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -193,7 +193,7 @@ class ProjectView : RComponent<ProjectExecutionRouteProps, ProjectViewState>() {
 
         if (state.isFirstTypeUpload == true) {
             gitUrlFromInputField?.let {
-                val newGitDto = GitDto(url = it)
+                val newGitDto = GitDto(url = it, username = gitDto?.username, password = gitDto?.password)
```

Comment:
```suggestion
                val newGitDto = gitDto?.copy(url = it) ?: GitDto(url = it)
```
Why not use `copy` in case we add other important fields in the future?